.github/workflows/build.yml

```diff
@@ -0,0 +1,84 @@
+name: Build
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize]
+
```

Comment:
what about adding:
```
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true
```
?

Reply:
is it necesssary now? have only one workflow

Reply:
it cancels jobs in rpogress when new changes are pushed (So i think so).
And we have only one workflow but multiple jobs so I'll add it.